### DISCUSSION TITLE
Einstellungen Hardware lädt nicht

### DISCUSSION THREAD

**Discussion by Eisbier2000** [tbnobody/OpenDTU#1048] [General]
Am Anfang erstmal Danke für das Projekt und die Arbeit die Ihr reingesteckt habt. 

Ich wollte mein Display zum laufen bringen. Allerdings lädt die Seite Einstellungen- Hardware nicht. Sowohl unter ios als auch unter windows nicht. Ist das ein Problem, das nur ich habe?

**Comment by Abphil999** [comment]
Ich habe aktuell genau dasselbe Problem. Weder unter Windows noch Android habe ich Zugriff auf die Hardwareeinstellungen. Der Ladebalken dreht in Dauerschleife.

**Reply by Szerb** [reply]
Hatte ich auch mal. Bei mir war das json Konfig schrott! Einfach mal mit dem Editor ins File schauen, vor dem hochladen.

**Reply by Abphil999** [reply]
Danke, das war es tatsächlich. Die Anleitung unter [https://blog.helmutkarger.de/balkonkraftwerk-teil-8-opendtu-und-ahoydtu-fuer-hoymiles-wechselrichter/](url) verweist warum auch immer auf eine inkompatible .json. Habe jetzt die im Original-Ordnerverzeichnis genommen und siehe da - funktioniert! Vielen Dank!

**Comment by Eisbier2000** [comment]
Danke das hat zumindest punktuell geholfen. Ich habe alle Einträge mit CMT2300A rausgeschmissen. Damit lädt jetzt die Hardwareseite. Allerdings bekomme ich mein Display nicht zum laufen. 
Ausgewählt habe ich das Profil  "NRF24 with SSD1306". Ich hab ein Display IC2 SSD1306 white 0,91 Zoll. 
Die Pins sind verbunden:
ESP32     ->      SSD1306
GND                 GND
5V                     VCC
22                     SCL
21                     SDA

im Profil der DTU habe ich folgende Einträge:

Display | clk | 22 | 22
-- | -- | -- | --
cs |   | 255
data | 21 | 21
reset |   | 255
type | 2 | 2

keines dieser Einträge ist farblich unterlegt.

Zum Zeitpunkt des Testes produzierte der Wechselrichter Strom.

Habe ich etwas falsch gemacht? Kann ich das Display direkt testen, ob es fehlerhaft ist.

Vielen Dank für die Hilfe!
